main.py

```diff
@@ -20,18 +20,10 @@
 
 epitech_accounts = json.load(open('config.json'))
 
-def extractAutologin(s):
-    if s is None:
-        return None
-    m = re.search(r'(auth\-[\da-f]{40})', s)
-    if m:
-        return m.groups()[0]
-    return None
-
 def extractCalendarID(s):
     if s is None:
         return None
-    m = re.search(r'([\da-z]+@group.calendar.google.com)', s)
+    m = re.search(r'([\w\.]+@gmail.com)', s)
```

Comment:
Are you sure about this? Where the `@gmail.com` extension come from ?

These addresses are not e-mail addresses, but rather calendar addresses. They must come from your Google calendar, using the following steps:
<https://github.com/skyf0l/EPITECH_to_GOOGLE_calendar/blob/c5df277ee4246570c5f6fec8ec1c381c99f0c24c/README.md?plain=1#L98-L104>

See mine:
![image](https://github.com/skyf0l/EPITECH_to_GOOGLE_calendar/assets/59019720/a03f2cd3-48d1-4cec-bb9b-2d905eb8f385)

Can you explain more about this change please?

Reply:
The identifier of my agenda ends in @gmail, I said to myself that following a change at google, the recent agendas end in @gmail, it is perhaps a bad deduction on my part

![Capture d’écran_2024-03-08_23-21-10](https://github.com/skyf0l/EPITECH_to_GOOGLE_calendar/assets/77923554/bc434b48-1b0a-4359-a21c-f1d2be4b497c)

Reply:
Ok, I get it, it's because you're trying to sync the epitech calendar to your personal google calendar. The only calendar ending in `@gmail.com` is your personal calendar, the others are newly created calendars. I never paid attention to that because I always created new calendars for epitech activities and I can advise you to do that too (for better organization).

I think it would be preferable to leave the old extension everywhere but only update the regex, what do you think?

Reply:
Oh, I understand, yes, I'll create a specific calendar, so let's keep the old extension